src/TestSuite/Fixture/FixtureInjector.php

```diff
@@ -17,17 +17,19 @@
 loadPHPUnitAliases();
 
 use Cake\TestSuite\TestCase;
-use PHPUnit\Framework\BaseTestListener;
 use PHPUnit\Framework\Test;
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestListenerDefaultImplementation;
 use PHPUnit\Framework\TestSuite;
 
 /**
  * Test listener used to inject a fixture manager in all tests that
  * are composed inside a Test Suite
  */
-class FixtureInjector extends BaseTestListener
+class FixtureInjector implements TestListener
 {
 
+    use TestListenerDefaultImplementation;
```

Comment:
Does this trait exist in older versions of PHPUnit?

Reply:
Exists since 6.4

Reply:
CakePHP 3.x needs to be compatible with PHPUnit 5.7 and PHP 5.6.

You probably need to update your composer file to require PHPUnit 6.x as the core [composer.json](https://github.com/cakephp/cakephp/blob/master/composer.json#L44).

Reply:
ok, sorry rushing here